src/screen/RequestScreen.js

```diff
@@ -238,6 +235,18 @@ class RequestScreen extends Screen {
 	}
 
 	/**
+	 * Adds aditional data to the body of the request in case a submit button
+	 * is captured during form submission.
+	 * @param {!FormData} body The FormData containing the request body.
+	 */
+	maybeAppendSubmitButtonValue(body) {
+		const button = globals.capturedFormButtonElement;
+		if (body && button && button.name && !button.disabled) {
```

Comment:
I added that as a safety measure to conform with the spec, but you're right, It shouldn't be needed. Let me check with that works across all browsers.